prime-router/metadata/fhir_mapping/hl7/resource/DiagnosticReport.yml

```diff
@@ -274,4 +274,18 @@ extension_6:
       constants:
         system: "http://terminology.hl7.org/CodeSystem/v2-0203"
         code: "FILL"
-        display: "Filler Identifier"
\ No newline at end of file
+        display: "Filler Identifier"
```

Comment:
This might be relevant to some comments in the OBR PR, we probably want to align on whether OBR and ORC should both have this display field, or neither. Maybe a 16th/17th minute for Monday.